app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Patient Details</p>
+  </div>
+</section>
+
+  <section class="section">
+    <div class="container">
+      <p class="title">FHIR Bundle</p>
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">FHIR Bundle</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+            <div class="content">
+              <%= @health_card.to_json %>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
+    <p class="title">Minified FHIR Bundle</p>
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">Minified FHIR Bundle</p>
+        <a class="card-header-icon card-toggle">
+          <i class="fa fa-angle-down"></i>
+        </a>
+      </header>
+        <div class="card-content">
+          <div class="content">
+           <pre>
+              <%= @health_card.bundle.to_json %>
+           </pre>
+          </div>
+        </div>
+      </div>
+    <br>
+    <br>
+    <br>
+
+    <p class="title">Encoded JWS</p>
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">Encoded JWS</p>
+        <a class="card-header-icon card-toggle">
+          <i class="fa fa-angle-down"></i>
+        </a>
+      </header>
+        <div class="card-content">
+          <div class="content", style="word-wrap: break-word; width: 800px">
+            <%= @jws_encoded_details%>
+          </div>
+        </div>
+      </div>
+    <br>
+    <br>
+    
+    <p class="title">QR Details</p>
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">QR Details</p>
+        <a class="card-header-icon card-toggle">
+          <i class="fa fa-angle-down"></i>
+        </a>
+      </header>
+        <div class="card-content">
+          <div class="content", style="word-wrap: break-word; width: 800px">
+            <%= @qr_code_payload %>
```

Comment:
I think the entire qr code payload, including the prefix, should be displayed here. Also, they shouldn't be displayed as raw arrays.

Reply:
resolved